src/ui.rs

```diff
@@ -198,11 +199,18 @@ pub fn ui<B: Backend>(f: &mut Frame<B>, app_state: &mut AppState) {
                                 ),
                             ])
                         } else {
-                            Spans::from(vec![Span::raw(format!(
-                                "{}. {}",
-                                i + 1,
-                                choice.content.to_string()
-                            ))])
+                            Spans::from(vec![Span::styled(
+                                format!("{}. {}", i + 1, choice.content.to_string()),
+                                if let Some(value) = card.correct_answer {
+                                    if value {
+                                        Style::default().fg(Color::Green)
```

Comment:
how can we create a global styles so that if we decide that we want a different theme someone can easily change it!